clients/csrf/transport.go

```diff
@@ -1,14 +1,16 @@
 package csrf
 
 import (
-	"net/http"
-
 	"github.com/jinzhu/copier"
+	"net/http"
 )
 
 type Csrf struct {
-	Header string
-	Token  string
+	Header              string
+	Token               string
+	IsInitialized       bool
```

Comment:
Is this needed?

Reply:
Yes, you need to know if the csrf is initialized or not.